app/src/components/market/sections/market_list/market_home.tsx

```diff
@@ -311,7 +313,9 @@ export const MarketHome: React.FC<Props> = (props: Props) => {
 
   useEffect(() => {
     setShowAdvancedFilters(
-      (currentFilter.currency || currentFilter.arbitrator || currentFilter.marketValidity === MarketValidity.INVALID) &&
+      (currentFilter.currency ||
+        currentFilter.arbitrator ||
+        currentFilter.curationSource === CurationSource.ALL_SOURCES) &&
         !fetchMyMarkets,
     )
```

Comment:
We want the advanced filters to stay open when you select a dropdown item, but also not necessarily be open on page load. You can emulate the following code:

```
const isInitialMount = useRef(true)
useEffect(() => {
  if (isInitialMount.current) {
    isInitialMount.current = true
  } else {
    setShowAdvancedFilters(
      (currentFilter.currency || currentFilter.arbitrator || currentFilter.marketValidity) && !fetchMyMarkets,
    )
  }
}, [currentFilter, fetchMyMarkets])
```